plugin/src/main/java/io/snyk/eclipse/plugin/runner/ProcessRunner.java

```diff
@@ -66,20 +73,26 @@ public ProcessResult run(ProcessBuilder pb, Optional<File> navigatePath) {
       return new ProcessResult(content.toString(), error.toString());
 
     } catch (IOException e) {
-      e.printStackTrace();
+      SnykLogger.logError(e);
       return new ProcessResult("", e.getMessage());
     }
   }
 
-  public ProcessBuilder createLinuxProcessBuilder(String command, Optional<String> path) {
+  public ProcessBuilder createLinuxProcessBuilder(List<String> command, Optional<String> path) {
     return getProcessBuilder(command, path, DEFAULT_LINUX_PATH);
   }
 
-  private ProcessBuilder getProcessBuilder(String command, Optional<String> path, String defaultLinuxPath) {
-    ProcessBuilder pb = new ProcessBuilder("sh", "-c", command);
+  private ProcessBuilder getProcessBuilder(List<String> command, Optional<String> path, String defaultPathForOS) {
```

Comment:
Is this `List<String> params`, instead of commands ?

Reply:
it actually is - I will update the naming.